docs/products/redis/howto/estimate-max-number-of-connections.rst

```diff
@@ -0,0 +1,23 @@
+Estimate maximum number of connection in Aiven for Redis
```

Comment:
Should we just say: "Estimate maximum number of connections" you're in the Redis subtree so I would keep the title shorter

Reply:
Agree!